account_financial_report/wizard/general_ledger_wizard_view.xml

```diff
@@ -64,6 +64,13 @@
                                 />
                             </group>
                         </page>
+                        <page string="Filter account types">
```

Comment:
This shouldn't go into a new tab, but on the same one, as the filtered accounts are on the other tab.

Reply:
> This shouldn't go into a new tab, but on the same one, as the filtered accounts are on the other tab.

Done

![image](https://github.com/OCA/account-financial-reporting/assets/69807420/c3d1fb8f-b1cc-4c7f-970d-964728662b8d)